app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,106 +10,105 @@
       name="isListExpanded"
       type="Boolean" />
 
-     <variable
+    <variable
       name="viewModel"
       type="org.oppia.android.app.hintsandsolution.HintsViewModel" />
   </data>
 
-    <LinearLayout
-      android:id="@+id/hint_summary_container"
+  <LinearLayout
+    android:id="@+id/hint_summary_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="32dp"
+    android:orientation="vertical">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginTop="32dp"
-      android:orientation="vertical">
+      android:layout_marginStart="72dp"
+      android:layout_marginEnd="72dp"
+      android:layout_marginBottom="20dp"
+      android:orientation="horizontal">
 
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
+      <TextView
+        android:id="@+id/hint_title"
+        style="@style/Heading2"
+        android:layout_width="wrap_content"
```

Comment:
Attributes like `layout_width`, `layout_height` and `textColor` can also be removed as they are getting enforced by the style. 
Remember, the style `Heading1` has a parent `TextView.Common1` which also enforces some attributes. Ditto elsewhere.

Reply:
Done